include/TrackContainer.h

```diff
@@ -41,6 +41,12 @@ class EXPORT TrackContainer : public Model, public JournallingObject
 {
 	Q_OBJECT
 public:
+	enum TrackContainerTypes
+	{
+		BBTrackContainer,
```

Comment:
Our wiki page says to use `CamelCase`, which is also what Qt uses, so I suggest we stick with it. I'd expect `UPPER_CASE` names to be #defines.